tests/api/ils/test_patrons.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test patrons endpoint."""
+
+import json
+
+from flask import url_for
+
+from tests.helpers import user_login
+
+
+def _patron_loans_request(client, json_headers, document_pid):
+    """Perform a patron loans request."""
+    params = []
+    params.append("document_pid={}".format(document_pid))
+    response = client.get(
+        "{}?{}".format(
+            url_for("invenio_app_ils_patrons.get_user_information"),
+            "&".join(params),
+        ),
+        headers=json_headers,
+    )
+    return json.loads(response.data.decode("utf-8"))
+
+
+def _assert_patron_loans_information(client, json_headers, document_pid,
+                                     expect):
+    """Assert patron loan information."""
+    resp = _patron_loans_request(client, json_headers, document_pid)
+    assert resp == expect
+
+
+def _assert_error_when_not_authenticated(client, json_headers,
+                                         document_pid, expect):
+    """Assert user is not logged in."""
+    resp = _patron_loans_request(client, json_headers, document_pid)
+    if resp['status'] == 401:
```

Comment:
I would just change this to:
```suggestion
    assert res.status_code == 401
```
and remove the `expect` param in this case.
In general, if and whenever possible, I would avoid `if` as much as possible in tests.